packages/realm-web/src/User.ts

```diff
@@ -34,12 +34,12 @@ const DEFAULT_DEVICE_ID = "000000000000000000000000";
 type SimpleObject = Record<string, unknown>;
 
 interface HydratableUserParameters {
-  app: App<any>;
+  app: App<any, any>;
   id: string;
 }
 
 interface UserParameters {
-  app: App<any>;
+  app: App<any, any>;
```

Comment:
Why were these lines changed?  Was there a typing issue here?

Reply:
Yes.  I followed the build process in twice, once as a commit following tag realm-web-v1.3.0 & this was not a problem.  Problem occurred when I cherry-picked my commit onto master branch.  My build process both times was git ```submodule update --recursive; npx lerna bootstrap``` before running build in dir packages/realm-web.  (I did not get past the cmake requirement to do a full build at top-level of the monorepo.)

Reply:
09:31 ~/Code/realm-js >node -v; npm -v
v16.5.0
7.19.1